### PR TITLE
LPD-20069 Fixed poshi test UseADTInSearchResultsWidget

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/LPD-20069

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase [hunk 1]
```diff
@@ -1231,6 +1231,8 @@ definition {
 			locator1 = "CommerceAccelerators#SORT_BY_DROPDOWN",
 			value1 = "Sort By: Relevance");
 
+		var productId = ${productId} - 1;
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase:1234]: We've spoken offline, please use data-qa-id

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase]: Did it, thanks!

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:forward:force

**Pull request closed without merging**